frontend/src/app/oe/services/oe-energy.service.ts

```diff
@@ -504,4 +504,15 @@ export class OeBlocktimeApiService {
   ): Observable<PaginationResponse<BlockTimeStrikeGuessPublic>> {
     return this.$strikesGuessesWithFilter( pageNo, { ... filter, class: 'outcomeKnown'} );
   }
+
+  $getPastStrikesPaginationWithFilter(
```

Comment:
there is a `strikesWithFilter` API call now instead of this one

Reply:
Done